examples/mdetr/model.py

```diff
@@ -0,0 +1,370 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from collections import OrderedDict
+from functools import partial
+from typing import Any, Dict, List, NamedTuple, Optional
+
+import torch.nn.functional as F
+from examples.mdetr.loss import (
+    contrastive_alignment_loss,
+    masked_dict_accuracy,
+    masked_dict_cross_entropy,
+    MDETRLoss,
+)
+from examples.mdetr.matcher import HungarianMatcher
+from torch import nn, Tensor
+from torchmultimodal.models.mdetr import MDETR, mdetr_resnet101, MDETRModelOutput
+from torchmultimodal.modules.losses.mdetr import box_losses, soft_token_prediction_loss
+
+
+class MultiHead(nn.Module):
```

Comment:
do u really need this? isnt moduledict enough?

Reply:
Yeah maybe, lemme see about replacing this in a follow-up